src/utils/dual_results.jl

```diff
@@ -23,9 +23,9 @@ end
 
 # internal function to parse through the reference dictionary and grab the file paths
 function _reading_references(results::Dict, duals::Array, stage::String, step::Array,
```

Comment:
_read_references* In the future prefer to use plain form verbs instead of the present participle.